inlong-sort-standalone/pom.xml

```diff
@@ -59,6 +59,7 @@
         <mockito.version>2.23.0</mockito.version>
         <jakarta.validation.version>2.0.2</jakarta.validation.version>
         <elasticsearch-version>6.8.13</elasticsearch-version>
+        <cls-version>1.0.5</cls-version>
```

Comment:
The same style “xxx.version” is recommended

Reply:
thx, fixed